tests/L0/run_fused_layer_norm/test_fused_layer_norm.py

```diff
@@ -8,291 +8,282 @@
 from apex.normalization import MixedFusedLayerNorm
 from apex.normalization import MixedFusedRMSNorm
 
+from torch.testing._internal import common_utils
+from torch.testing._internal.common_device_type import instantiate_device_type_tests
+from torch.testing._internal.common_device_type import onlyCUDA
 
-class TestFusedLayerNorm(unittest.TestCase):
-    dtype = torch.float
-    elementwise_affine = False
-    normalized_shape = [32, 16]
-    rtol, atol = None, None
-    fwd_thresholds = dict(rtol=None, atol=None)
-    bwd_thresholds = dict(rtol=None, atol=None)
-    mixed_fused = False
-
-    def setUp(self):
-        # bias and weight are set to 0 and 1 respectively, so no need to copy parameters from cpu module to the gpu one
-        if not self.mixed_fused:
-            self.module_cpu_ = FusedLayerNorm(
-                normalized_shape=self.normalized_shape, elementwise_affine=self.elementwise_affine).cpu()
-            self.module_cuda_ = FusedLayerNorm(
-                normalized_shape=self.normalized_shape, elementwise_affine=self.elementwise_affine).to(device="cuda", dtype=self.dtype)
-        else:
-            assert self.elementwise_affine
-            self.module_cpu_ = MixedFusedLayerNorm(
-                normalized_shape=self.normalized_shape).cpu()
-            self.module_cuda_ = MixedFusedLayerNorm(
-                normalized_shape=self.normalized_shape).to(device="cuda", dtype=self.dtype)
+from itertools import product
```

Comment:
`itertools` is imported above, so could you remove either of them?

Reply:
Fixed